source/code/plugins/operation_lib.rb

```diff
@@ -39,9 +39,28 @@ def filter(record, time)
 
       return {}
     end
+
+    def filter_generic(record,time)
+      if record.is_a?(Hash) && !record.empty? && record.has_key?("message")
+        dataitem = {}
+        dataitem["Timestamp"] = OMS::Common.format_time(time)
+        dataitem["OperationStatus"] = "Warning"
+        dataitem["Computer"] = OMS::Common.get_hostname or "Unknown host"
+        dataitem["Detail"] = record["message"]
+        return dataitem
+      end
+      return {}
+    end
  
-    def filter_and_wrap(record, time)
-      data_item = filter(record, time)
+    def filter_and_wrap(tag, record, time)
+      tag_type = tag.sub(/^oms.operation./,'')
```

Comment:
can we use the last part of the tag as the tag_type? don't assume that it starts with oms.operation.